packages/linter/src/rules/NoIconFontIsUsed.ts

```diff
@@ -32,7 +32,7 @@ const ICON_FONT_IDENTIFIERS = [
   },
   {
     className: "icon-",
-    fontFamilies: ["icon"],
+    fontFamilies: ["icon", "icons", "icomoon"],
```

Comment:
Can you add a test case? Have we already got tests for all the icon font providers?